openapi_python_client/parser/properties/__init__.py

```diff
@@ -155,6 +159,13 @@ class UnionProperty(Property):
 
     inner_properties: List[Property]
     template: ClassVar[str] = "union_property.pyi"
+    has_properties_without_templates: bool = attr.ib(init=False)
+
+    def __attrs_post_init__(self) -> None:
+        super().__attrs_post_init__()
+        object.__setattr__(
+            self, "has_properties_without_templates", any(prop.template is None for prop in self.inner_properties)
+        )
```

Comment:
Any reason for doing it this way vs. doing
```python
@property
def has_properties_without_templates(self) -> bool:
    return any(prop.template is None for prop in self.inner_properties)
```
?

I guess this way may be a bit more efficient as its only being computed once

Reply:
The efficiency is exactly why. In the Template we need to reference this for each property to know whether or not we need an else, so it can get expensive to have to loop every time.

Reply:
Ah gotcha, definitely better this way then haha